README.md

```diff
@@ -232,14 +232,16 @@ az login
 az aks list -o tsv --query '[].resourceGroup'
 
 # pins-uks-k8s-dev
+# pins-uks-k8s-preprod
 # pins-uks-k8s-prod
 ```
 
 ```shell script
 az aks list -o tsv --query '[].name'
 
-# pins-uks-k8s-1234-dev
-# pins-uks-k8s-1234-prod
+# pins-uks-k8s-5883-dev
```

Comment:
These may change - might be worth putting a note to that effect

Reply:
OK.